examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/mix-precision/README.md

```diff
@@ -0,0 +1,65 @@
+# Run
+ 
+In this examples, you can verify the accuracy on HPU/CUDA device with emulation of MXFP4, MXFP8, NVFP4 and uNVFP4.
+
+## Requirement
+
+```bash
+# neural-compressor-pt
```

Comment:
why using INC personal branch here, suppose user need reproduce it with release version

Reply:
You're right, this branch is for the PR and will be deleted.